### PR TITLE
Fix SQL delay in all the queries/stored proce

### PR DESCRIPTION
**Pull request by cijothomas**
Continuation of this:
https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/234

Added SQL delay in all the queries. Above PR modified only 1 query.

### CODE REVIEW

File: Test/DependencyCollector/FunctionalTests/FuncTest/IIS/IisApplicationPool.cs [hunk 1]
```diff
@@ -96,6 +96,7 @@ public IisApplicationPool(string poolName, string managedRuntimeVersion = Defaul
             catch (Exception ex)
             {
                 Trace.TraceError("Exception occured while creating IIS Application Pool: {0}. The exception thrown was {1}", poolName, ex.Message);
+                throw ex;
```
Comment (cijothomas) [inline Test/DependencyCollector/FunctionalTests/FuncTest/IIS/IisApplicationPool.cs]: This was meant to be a diff PR as this was unrelated. but i did not change branches while making this change.

Comment (cijothomas) [inline Test/DependencyCollector/FunctionalTests/FuncTest/IIS/IisApplicationPool.cs]: btw - this is simply a fix to makes tests fail instantly if initialize has issues

Comment (dnduffy) [inline Test/DependencyCollector/FunctionalTests/FuncTest/IIS/IisApplicationPool.cs]: Just use "throw;" so you don't lose the stack trace.

Comment (cijothomas) [inline Test/DependencyCollector/FunctionalTests/FuncTest/IIS/IisApplicationPool.cs]: will modify.

**Pull request merged**